dev/com.ibm.ws.microprofile.config_fat_tck/fat/src/org/eclipse/microprofile/config/tck/ConfigTckPackageTest.java

```diff
@@ -49,22 +45,8 @@ public static void tearDown() throws Exception {
     @Test
     @AllowedFFDC // The tested deployment exceptions cause FFDC so we have to allow for this.
     public void testConfigTck() throws Exception {
-        if (!MvnUtils.init) {
```

Comment:
I had a version like yours but I chose to undo it as I did not want to standardise all the directory/file names in the common code in a way that made it harder to have multiple  TCK FAT tests (results directories) co-oexisting in one FAT bucket. That is why these file names are here in the FAT in a single test and not in MvnUtils. I think what I had (and what you have done) looks neater and is easier to clone though. So I will approve it.